src/styles/blocks/main.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  @extend %flex;
+  flex-wrap: wrap;
+  max-width: getDaysWidth($days-block);
+  padding: 50px 0 50px $block-padding-left;
+  margin: 0 auto;
```

Comment:
you can`t use margins for BEM blocks

Reply:
Fix your mate's comment. Try to center the calendar in another way.